### ISSUE TITLE
Removing/Reworking suggestions about orcs eating other sapient races from mainline

### ISSUE THREAD

**Issue by Konrad22** [wesnoth/wesnoth#4007]
As was pointed out on the WoV [forum thread](https://forums.wesnoth.org/viewtopic.php?p=640479#p640479), there are multiple suggestions/references about orcs eating other sapient races, which is, well, disturbing.
I am with @CelticMinstrel in being interested in seeing those (and others if there are more) removed.

The lines in question:

["Smells like lots of death in the south. There must be a good feast near that merman town."](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/Dead_Water/scenarios/03_Wolf_Coast.cfg#L210)

["Ah, another human is approaching... yes... more meat for my war-band..."](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/Delfadors_Memoirs/scenarios/04_Leollyn.cfg#L165)

["While I have no love for it, without the help of the dead we consign ourselves and our families to orcish stewpots!"](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/Descent_Into_Darkness/scenarios/01_Saving_Parthyn.cfg#L227)

["Hey look, there is our meat! Come on, boys, lunchtime!"](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/Northern_Rebirth/scenarios/03_To_the_Mines.cfg#L117)

["How’d they get behind us? I hate to miss such tasty meat."](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/The_Rise_Of_Wesnoth/scenarios/04b_The_Midlands.cfg#L310)

["... orcs are not going to tolerate living humans outside of the slave mines or stew pot!"](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/The_Rise_Of_Wesnoth/scenarios/07_Return_to_Oldwood.cfg#L299)

["Hmm, they left off the ‘or the orcs will eat you’ part."](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/The_Rise_Of_Wesnoth/scenarios/20_Return_of_the_Fleet.cfg#L366)

["I hate it when my prey gets chatty!"](https://github.com/wesnoth/wesnoth/blob/1.14.6/data/campaigns/The_Rise_Of_Wesnoth/scenarios/22_The_Rise_of_Wesnoth.cfg#L360)

(That last one is rather ambigous and I wouldn't think that it refers to eating Humans unless presented together with such an array of quotes.)

**Comment by CelticMinstrel** [comment]
I wonder if the DiD one could be left in as a human misunderstanding, but that aside, I definitely don't like this implication about orcs.

**Comment by nemaara** [comment]
Actually, I want to hold off on this. If it does turn out that WoV convincingly does the "drakes at first eat humans" narrative, we can make the "orcs eat humans" work as well (it will require some extra writing, can't be left as is).

**Comment by sigurdfdragon** [comment]
I disagree with removing/changing the orc references listed in the OP, for at least the following:
1. Orcs are the bad guys.
2. Four of the lines are from TRoW. As it seems to be regarded as a well-written campaign, it would be inappropriate to just adjust the dialog without considering how it affects TRoW.
3. Note that in Lord of the Rings: The Two Towers (movie) there is a scene showing that the Uruk-hai grunts want to eat the hobbits Pippin and Merry.

**Comment by Ja-MiT** [comment]
It might be worth being more discerning about these quotes. It seems to me that they are not equally problematic. (For example, as CelticMinstrel noted, popular misconceptions are not unheard of.) I think most of these lines could be kept because there is little reason to assume they are truthful and accurate. Change two, maybe three, of them.

A better idea than rewriting or deleting dialog might be to add some. Use SotBE to sort out the myths and misconceptions about orcs. Have a scene where an underling asks something like "we don't really have to eat them, do we?" Or whatever the writers can come up with.

**Views by others**
Other races view orcs as bloodthirsty, savage villains. Eating people is evil. So orcs probably do it (from the other races' perspective). Rumors can be more compelling than facts.
* "While I have no love for it, without the help of the dead we consign ourselves and our families to orcish stewpots!"
* "... orcs are not going to tolerate living humans outside of the slave mines or stew pot!"
* "Hmm, they left off the ‘or the orcs will eat you’ part."

**Trash talk**
The misconceptions popular among the other races provide orcs with an opportunity for intimidation. I forget the precise setting for these quotes, but if they are directed at -- or intended to be overheard by -- the good guys, then they could be [bark instead of bite](https://en.wiktionary.org/wiki/one%27s_bark_is_worse_than_one%27s_bite). (It becomes a judgement call as to what goes over the line.)
* "Ah, another human is approaching... yes... more meat for my war-band..."
  * _(hmm... changing "war-band" to "wolf pack" might work)_
* "Hey look, there is our meat! Come on, boys, lunchtime!"
* "I hate it when my prey gets chatty!"

**Eating**
At some point, the quotes do cross over into strongly suggesting that orcs hunt people for food. Perhaps more trash talk, but perhaps not. Establishing that orcs do eat people might be over the top, and perhaps the campaigns could be improved with some touch-up re-writing. ("Might" because, well, orcs do mostly play the role of the evil bogyman who will eat you in your sleep. Still, I don't think much story is lost if these lines are changed.)
* "Smells like lots of death in the south. There must be a good feast near that merman town."
  * _(This line looks the most problematic to me, and it's possibly the easiest to replace.)_
* "How’d they get behind us? I hate to miss such tasty meat."
  * _(While this could be trash talk, the initial question suggests to me that it is not intended by the speaker to be overheard.)_

**Comment by CelticMinstrel** [comment]
> Orcs are the bad guys.

I strongly disagree with this as a general principal. Depicting them as just "the bad guys" is not good writing. No species does "bad things" just to be bad. Orcs are a different culture, maybe a warrior culture that places less value on the life of individuals, for example. And the Son of the Black-Eye campaign already exists as a counterpoint to the narrative of "orcs as bad guys"... okay, you might argue that that's a campaign where you play as the bad guys, but from what I've played of it, it didn't really feel like that. (I got like, one or two scenarios past the appearance of the shamans.)

> Note that in Lord of the Rings: The Two Towers (movie)

Wesnoth is not Lord of the Rings. In Lord of the Rings, the orcs are actually elves that have been corrupted by the power of Morgoth; and the Uruk-Hai are a magical mix between orcs and humans. In Wesnoth, orcs are an entirely separate species. So any argument from the basis of Lord of the Rings is meaningless.

@Ja-MiT – Thanks for the detailed analysis! I agree, they are not all equally problematic, and of course they have to be taken in context. I'll note that the first two lines in your "trash talk" section are clearly not directed at humans, though I can't say whether they're intended to be overheard by humans. For the third, "prey" has a weaker implication of eating, so at least when taken out of context, it's not as bad as the others.

The DW line could easily be changed to replace "feast" with something like "battle", or maybe something with an implication of looting.

**Comment by Wedge009** [comment]
Some deep thinking in everyone's posts here. I admit I didn't really think about orcs eating humans until I saw the quotes konrad listed. Honestly, I think I took them in the way that Ja-MiT lists as 'intimidation talk' rather than an actual statement of 'we are hungry!' I've heard many a time in movies where a villainous character might say something like 'ah, fresh meat!' regarding an opponent they expect to defeat very easily, and it's pretty clear they don't mean to actually eat their victim.

In other words, with how I've ready Wesnoth lore up to now I don't think orcs eat humans, at least not as a normal, regular source of food.

**Comment by sigurdfdragon** [comment]
> "Ah, another human is approaching... yes... more meat for my war-band..."

As the current maintainer of DM, I'm keeping this line as is currently.

> Depicting them as just "the bad guys" is not good writing

Doesn't make it bad writing. Sometimes things can just be simple.

> So any argument from the basis of Lord of the Rings is meaningless.

Disagree. Intended that as an illustration of what's being done/considered acceptable in the broader culture-scape. And it is possible/somewhat likely that the LotR movies, having come out before BfW, influenced some of the lines in question.

> And the Son of the Black-Eye campaign already exists as a counterpoint to the narrative of "orcs as bad guys"... okay, you might argue that that's a campaign where you play as the bad guys, but from what I've played of it, it didn't really feel like that.

It doesn't feel like playing the bad guys. All the campaigns in mainline that have you play 'the bad guys' spin it so there is some amount of relatable/they are not really evil. It doesn't have to be that way.

**Comment by CelticMinstrel** [comment]
> Doesn't make it bad writing. Sometimes things can just be simple.

It's _possible_ for things to be simple, but the orcs in Wesnoth don't appear to fit that description.

> Disagree. Intended that as an illustration of what's being done/considered acceptable in the broader culture-scape. And it is possible/somewhat likely that the LotR movies, having come out before BfW, influenced some of the lines in question.

I'm kinda peeved that you're calling out the movies specifically, but that aside... the issue is less whether it's "acceptable" and more whether it "makes sense in context". For the orcs in Lord of the Rings, there is an explanation for the behaviour. I don't see any such thing for the orcs in Wesnoth, and the LotR explanation clearly doesn't apply.

> It doesn't feel like playing the bad guys. All the campaigns in mainline that have you play 'the bad guys' spin it so there is some amount of relatable/they are not really evil.

You call it a "spin" but that doesn't make the campaign any less "canonical". Indeed, anything that SotBE says about orcs should probably take priority over anything other campaigns say (where they contradict sufficiently), with the stuff from other campaigns being the misconceptions of other races. The very existence of this campaign with its more relatable "spin" is proof that the orcs cannot be _just_ bad guys.

> It doesn't have to be that way.

I'm not even sure what you're trying to say with this bit...

**Comment by Wedge009** [comment]
Regarding orcs being evil or not-so-evil: given that Wesnoth players are most likely to be human, I think they would relate most to the human characters in the story. And orcs are often 'the bad guys' by default in Wesnoth stories and also from the genre's design, of which Lord of the Rings is no doubt a big influence. Nonetheless, there's no reason that Wesnoth orcs have to follow Lord of the Rings orcs exactly (I've seen some comments that Wesnoth perhaps copies Lord of the Rings too closely), and I like how the 'bad guys' campaigns give something relatable to play for and that they're not just stereotypically causing trouble for everyone just for the fun of it: Kapou'e seeking vengeance against a treacherous human and a traitorous orc, Malin seeking power to protect his home, Ardonna being 'misunderstood' in her quest for immortality, and Baldras leading his band of outlaws against a tyrant queen. It seems to me that that is the point in making the 'bad guys' campaigns not feel like 'playing as the bad guys' - while there might be some players who want to play as evil characters and revel in it, I think it's good that there is justification for these 'bad' factions' motives in their respective campaigns.

So... yes, most of the time orcs are depicted as being 'the bad guys' and that may well be justified. But I've still yet to see anything in current Wesnoth lore that clearly and unambiguously shows orcs as using humans as a regular food source. Coming back to the original topic for this issue, I think it'd be fair to clarify some orcs lines that might suggest they do so, but if people want to leave them as-is, that can be an option too. As I often see in creative decisions, letting the reader/viewer/player make up their own mind about how something works in a story may be the best outcome for everyone.

**Comment by MetaKnight** [comment]
I say leave it. This is kind of being a hypocrite- it's okay for other races to be eaten, but not humans. What makes us so special. I'd say humans are generally selfish troublemakers- in Wesnoth and in real life. If other race can get eaten, what's wrong with us?

**Comment by Wedge009** [comment]
I don't think it's specifically humans that's the issue, it's the eating of any sapient race. Although, eating of humans is perhaps the most common example because, let's face it, the average Wesnoth scenario writer is human and most of the cited examples above are with reference to humans.

**Comment by CelticMinstrel** [comment]
It's not okay for merfolk to be eaten. It's not okay for drakes to be eaten. It's not okay for saurians to be eaten. It's not okay for whales to be eaten.

This isn't something exclusive to humans.

**Comment by Wedge009** [comment]
Are there sapient whales in Wesnoth? I don't remember...

**Comment by nemaara** [comment]
I fixed the DiD one in 91738cc. Waiting for WoV to address the others (will add some text supporting orcs eating humans if WoV decides to keep the drake stuff, or will tweak some lines if that gets removed).

**Comment by Ja-MiT** [comment]
@nemaara I think the DiD quote actually became more fitting after establishing that orcs do not eat people. Think about it. Malin is young, brash, full of zeal, and suddenly in need of justification. Rational thought tends to disappear in a situation like that. Any justification would do, even if it means using an "[old wives' tale](https://en.wiktionary.org/wiki/old_wives%27_tale)" as if it was fact.

If orcs eat people, Malin's old line showed just a normal fear of a real danger. If orcs do not eat people, Malin's old line went into hyperbole, showing a mind that was trying to cope with a seemingly impossible situation. That fits well with Malin's later obsession with revenge on the orcs. The line had more drama before the fix.

**Comment by nemaara** [comment]
Not really after I rewrote DiD, it felt out of character. Anyway, it's just a small detail and not really worth worrying about. Malin's character is hardly based around that.

**Comment by nemaara** [comment]
Since WoV has been merged, I'm handling some of these lines now. See #4060.

**Comment by nemaara** [comment]
#4060 merged.

**Comment by Arcanister** [comment]
I think that if something is deemed bad in real life, such as killing people, eating them or something like that, it doesn't mean that it has to be removed from a fantasy book or game.

**Comment by CelticMinstrel** [comment]
...well, that's obvious.

That said, it also doesn't mean a given book or game's portrayal of it is acceptable and worth _not_ removing. Just saying.
